### PR TITLE
New package: kde5-5.8

### PR DESCRIPTION
**Pull request by dutsky**
Last part of KDE Plasma with kde5-metapackage, previous is #5849

### CODE REVIEW

File: srcpkgs/kde5/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	plasma-nm
+	plasma-pa
+	plasma-sdk
+	plasma-workspace-wallpapers
+	powerdevil
+	qt5-tools
+	sddm-kcm
+	user-manager
+	xorg"
+short_desc="The kde 5 meta-package for Void Linux"
```
Comment (Vaelatern) [inline srcpkgs/kde5/template]: `KDE` should mirror the GNOME meta-package for Void Linux.

**Comment by Vaelatern** [comment]
Looks acceptable to me. I'll merge in about 24 hours.

**Comment by Vaelatern** [comment]
oh, wait: `=> ERROR: xbps-src: unexistent file: /void-packages/srcpkgs/plasma-workspace-devel/template` this needs to be fixed. Travis wasn't wrong in this.

**Comment by silvernode** [comment]
Watching is this religiously, thanks for the effort.

**Comment by dutsky** [comment]
@Vaelatern it happens because libqalculate updated. I hope revbump of plasma-worspace will helps. I'd made PR for it: #7597
Let's hope it help, because it building fine on my machine

**Comment by silvernode** [comment]
This is installing right now, I just got home. I will report any issues I find. Thanks for all the hard work again. Woot!

**Pull request merged**